File: vol-100/01-cubic/cubic.cpp
#include <iostream>
using namespace std;

int cubic(int num)
{
	return num * num * num;
}

int main()
{
	int num;
	cin >> num;
	if( !cin )
	{
		return 1;
	}
	cout << cubic(num) << endl;
	return 0;
}

File: vol-100/01-cubic/Cubic.scala
import scala.io.Source

object Cubic
{
  def main(args: Array[String])
  {
    val num = readInt()
    println(cubic(num))
  }

  private def readInt() : Int =
  {
    val line = readLine()
    Integer.parseInt(line)
  }

  private def cubic(num:Int) : Int =
  {
    num * num * num
  }
}
